docs/aws_databases.md

```diff
@@ -0,0 +1,38 @@
+# AWS Database Options
+
+We store blobs and identify them by the digest. That means that if we were to use a schema-based database, the schema would be pretty simple - but the blobs can get quite big.
```

Comment:
blobs size is not an issue. we only store the blob digest as key and and its path as value, or I'm misunderstanding something.

Reply:
It was a misunderstanding on my part when I wrote the document. I will update it